app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -270,9 +272,11 @@ public void onConnected(Bundle bundle) {
     }
 
     private void initSummitNotificationHandler() {
-        NotificationHandler notificationHandler = new NotificationHandler(this);
-        if (notificationHandler.shouldSetAlarm()) {
-            notificationHandler.setAlarmForNotification();
+        for (TaggedEventSeries eventSeries : App.getInstance().currentTaggedEventSeries()) {
```

Comment:
If there was already an alarms with the same intent the previous is cancelled. So we are save.
See https://developer.android.com/reference/android/app/AlarmManager.html#set%28int,%20long,%20android.app.PendingIntent%29

Yes, to move it to `App`